docs_source/content/server.md

```diff
@@ -16,6 +16,32 @@ Converge comes with a server that can:
 - serve the Converge binary itself, for bootstrapping new systems inside your
   network
 
+## Basic Usage
+
+Install the converge binary to your server or device and run the `server` command.
+
+```shell
+converge server --rpc-token "YOLO security"
+```
+
+This will spin up a gRPC server on port 4774, with `YOLO security` set as the
+token string. You should see messages streaming from the server. If you run the
+server command without `--rpc-token`, then the output will include the
+generated token, and you can use that in place of `YOLO security` in future
+invocations.
```

Comment:
Could you mention that the token will be regenerated on every invocation of server if it's not set? It's worth emphasizing that this is really only for use with `--local`, although it does help get set up with a secured server quickly.